File: engine/engine-model/src/main/scala/tools/packet/party/PartySearchInvite.scala
package tools.packet.party

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class PartySearchInvite(private var _partyId: Int, private var _fromCharacterName: String) extends PacketInput {
  def partyId: Int = _partyId

  def fromCharacterName: String = _fromCharacterName

  override def opcode(): SendOpcode = SendOpcode.PARTY_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/party/PartySearchInvite.java
package tools.packet.party;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record PartySearchInvite(Integer partyId, String fromCharacterName) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.PARTY_OPERATION;
   }
}